nsds_lab_to_nwb/metadata/metadata_manager.py

```diff
@@ -187,6 +192,14 @@ def extra_cleanup(self):
         self.metadata_input['extra_meta'] = {}
         for key in ('block_meta', 'experiment_meta', 'other'):
             self.metadata_input['extra_meta'].update(self.metadata_input.pop(key, {}))
+        if block_meta['has_ecog']:
+            bad_chs = self.metadata_input['device']['ECoG']['bad_chs']
+            if len(bad_chs) > 0:
```

Comment:
Do you know what happens in the nwb file if there are no bad channels? IMO, it is nice to some explicit indication that there are no bad channels rather than having no info.

Reply:
I think it would be ideal if pynwb let us write an empty list

Reply:
Jesse's feedback: 

![image](https://user-images.githubusercontent.com/5800317/146102749-a2427af7-ea69-41fa-b3f9-4c1f6c6b9487.png)

Reply:
Looks like it would go here
https://github.com/BouchardLab/nsds_lab_to_nwb/blob/8ddd8f0b3ac639e1b3f07016b30e63efd8567c79/nsds_lab_to_nwb/components/electrode/electrodes_originator.py#L51

Reply:
@JesseLivezey Thanks for pointing that out. I added the relevant code. But now I get a TypeError when at the last step when the nwb file is being written to disk. I need to debug more. Have you ever run into this error?

```
Exception has occurred: TypeError
len() of unsized object
  File "/home/jhermiz/software/nsds_lab_to_nwb/nsds_lab_to_nwb/nwb_builder.py", line 258, in write
    nwb_fileIO.write(content)
  File "/home/jhermiz/software/nsds_lab_to_nwb/scripts/generate_nwb_john.py", line 63, in <module>
    nwb_builder.write(nwb_content)
```

Once I figure that out, then I would need to delete the code that adds `bad_chs` to scratch.

Reply:
TypeError is resolved. Feel free to review. Will merge tomorrow EOD.